src/main/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/repository/DuplicateOffenderRepository.java

```diff
@@ -69,4 +69,150 @@
             ")",
             nativeQuery = true)
     List<DuplicateOffender> getOffendersWithMatchingLidsNumbers(String offenderNo);
+
+    /**
```

Comment:
As a matter of interest what analytical research went into this to come up with the weightings etc?

Reply:
It wasn't hugely scientific, but using a subset of 157 offenders we ran through a few combinations of weightings to see at what point sensible duplicate suggestions would appear / disappear and tried to minimise false negatives over false positives. Some documentation here: https://dsdmoj.atlassian.net/wiki/spaces/NG/pages/1581940871/Finding+Potential+Duplicates+-+First+Draft

Reply:
In the midst of doing this we were told about the Analytical Platform linking solution so we didn't spend much more time developing this.  Really this is just to provide a bit of a comparison against the other tools to check they're working (and out performing this one).